From 35ad7482af49ab80d348b619b6d420ee170d3e5d Mon Sep 17 00:00:00 2001
From: Olivier Tilloy <olivier.tilloy@canonical.com>
Date: Mon, 8 Jan 2018 16:59:17 +0100
Subject: tdf#114915 Apparmor profile update (invalid comments syntax)

The apparmor profile parser doesn't interpret # as a comment when on the same
line as a variable assignment. Ensure that comments are on their own line,
above the actual assignment.

Change-Id: If10bb05f11b0e9026746db2e271113f30da5cbef
Reviewed-on: https://gerrit.libreoffice.org/47605
Tested-by: Jenkins <ci@libreoffice.org>
Reviewed-by: Thorsten Behrens <Thorsten.Behrens@CIB.de>
---
 sysui/desktop/apparmor/program.soffice.bin | 30 ++++++++++++++++++++----------
 1 file changed, 20 insertions(+), 10 deletions(-)

--- a/sysui/desktop/apparmor/program.soffice.bin
+++ b/sysui/desktop/apparmor/program.soffice.bin
@@ -21,12 +21,18 @@
 #Some obscure ones we're excluded (mostly input)
 
 #Generic
-@{libreoffice_ext} = [tT][xX][tT]  #.txt
-@{libreoffice_ext} += {,f,F}[oO][dDtT][tTsSpPbBgGfF]  #All the open document format
-@{libreoffice_ext} += [xX][mMsS][lL]  #.xml and xsl
-@{libreoffice_ext} += [pP][dD][fF]  #.pdf
-@{libreoffice_ext} += [uU][oO][fFtTsSpP]  #Unified office format
-@{libreoffice_ext} += {,x,X}[hH][tT][mM]{,l,L}  #(x)htm(l)
+#.txt
+@{libreoffice_ext} = [tT][xX][tT]
+#All the open document format
+@{libreoffice_ext} += {,f,F}[oO][dDtT][tTsSpPbBgGfF]
+#.xml and xsl
+@{libreoffice_ext} += [xX][mMsS][lL]
+#.pdf
+@{libreoffice_ext} += [pP][dD][fF]
+#Unified office format
+@{libreoffice_ext} += [uU][oO][fFtTsSpP]
+#(x)htm(l)
+@{libreoffice_ext} += {,x,X}[hH][tT][mM]{,l,L}
 
 #Images
 @{libreoffice_ext} += [jJ][pP][gG]
@@ -43,15 +49,19 @@
 
 #Calc
 @{libreoffice_ext} += [xX][lL][sSwWtT]{,x,X}
-@{libreoffice_ext} += [dD][iIbB][fF]  #.dif dbf
-@{libreoffice_ext} += [cCtT][sS][vV]  #.tsv .csv
+#.dif dbf
+@{libreoffice_ext} += [dD][iIbB][fF]
+#.tsv .csv
+@{libreoffice_ext} += [cCtT][sS][vV]
 @{libreoffice_ext} += [sS][lL][kK]
 
 #Impress/Draw
 @{libreoffice_ext} += [pP][pP][tTsS]{,x,X}
 @{libreoffice_ext} += [pP][oO][tT]{,m,M}
-@{libreoffice_ext} += [sS][wW][fF]  #Flash
-@{libreoffice_ext} += [pP][sS][dD]  #Photoshop
+#Flash
+@{libreoffice_ext} += [sS][wW][fF]
+#Photoshop
+@{libreoffice_ext} += [pP][sS][dD]
 
 #Math
 @{libreoffice_ext} += [mM][mM][lL]
